<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profilt Calculation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fee rates:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dbFeeRate">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="value">
         <double>0.000800000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Margin rates:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dbMarginRate">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dbCount">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Profilt Calculation:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Purchase price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePurchase">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Margin:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePurchasMargin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePurchaseFee">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Selling price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSelling"/>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Margin:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSellingMargin">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSellingFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Profilt:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leProfilt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leProfiltFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Shorts profit:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Selling price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShort_Selling"/>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Maring:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_SellingMargin">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_SellingFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Profilt:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_Profilt">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_ProfiltFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Purchase price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_Purchase">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Maring:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_PurchaseMargin">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leShorts_PurchaseFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Long profit:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Purchase price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_Purchase"/>
         </item>
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Margin:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_PurchaseMargin">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_PurchaseFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Profilt:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_Profilt">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_ProfiltFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Selling price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_Selling">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Margin:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_SellingMargin">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leLong_SellingFee">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dbFeeRate</tabstop>
  <tabstop>lePurchase</tabstop>
  <tabstop>leSelling</tabstop>
  <tabstop>leProfilt</tabstop>
  <tabstop>leSellingFee</tabstop>
  <tabstop>lePurchaseFee</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
